api/http/router.go

```diff
@@ -58,6 +60,10 @@ func NewRouter(app app.App) http.Handler {
 	router.Use(accesslog.Middleware())
 	// requestid attaches X-Men-Requestid header to context
 	router.Use(requestid.Middleware())
+	router.Use(identity.Middleware(
+		identity.NewMiddlewareOptions().
+			SetPathRegex(`^/api/management/v[0-9]/`),
+	))
```

Comment:
believe it or not but on my setup this is required, otherwise the identity is not propagated to the handler. For Alf it works, so if you think I should remove, I will remove it.

Reply:
@alfrunes ^^^